tests/main/views/test_direct_award.py

```diff
@@ -1691,3 +1696,66 @@ def test_direct_award_nonawarded_scenarios(
                     "none-suitable": "none-suitable",
                 }[endpoint_rel_path],
             }
+
+
+class TestDirectAwardProjectUpdate(DirectAwardSetupAndTeardown):
+
+    def project(self, direct_award_project_name):
+        project = DirectAwardProject(
+            name=direct_award_project_name,
+            users=[User.query.get(self.user_id)],
+        )
+        db.session.add(project)
+        db.session.flush()
+        return project
+
+    def test_project_update_endpoint_exists(self):
+        project = self.project(self.direct_award_project_name)
+        db.session.commit()
+
+        res = self.client.patch(f'/direct-award/projects/{project.external_id}',
+                                content_type='application/json',
+                                data=json.dumps({
+                                    'project': {},
+                                    'updated_by': 'example@example.com',
+                                }))
+
+        assert res.status_code == 200
```

Comment:
You could also assert here very cheaply that `stillAssessingAt` *wasn't* updated by this.

Reply:
I decided not to do that because it kind of goes against the 'test one thing' philosophy.

I see the value of testing that `stillAssessingAt` doesn't change though, so I will create a separate one that does this.

Reply:
Ok. Twice the running time and twice the volume of code to maintain though.

Reply:
(personally I haven't ever been convinced by the "test one thing" argument, and you're unlikely to see me insist on it)

Reply:
Done

Reply:
"Twice the running time" coming as a comment from the person who adds 100s to 1000s of tests at a time with parametrization? 😛

Reply:
And each of those would be 5 or 10 times longer if they didn't make multiple assertions per pass.